README.md

```diff
@@ -1 +1,34 @@
-# snet-converter-services
\ No newline at end of file
+# Snet Converter Services
```

Comment:
Please add some insights on the the repo.
1. What is it used for?
2. How can I run it locally to start developing with offline servlerless?